src/api/File.js

```diff
@@ -189,12 +189,18 @@ class File extends Item {
                 missingFields,
             );
 
+            // If fields are user fields, make sure if value is null to prepopulate
+            // with placeholder user
+            const filledInData = this.fillUserPlaceholder(
+                dataWithMissingFields,
+            );
+
             // Cache check is again done since this code is executed async
             if (cache.has(key)) {
-                cache.merge(key, dataWithMissingFields);
+                cache.merge(key, filledInData);
```

Comment:
I am also not sure if the cache should be having mocked out objects. I think the component should have defaults set on its end and not at the data layer. Since this assumes that mocked out placeholder is going to be used by all components.

Reply:
So if I'm reading you right, I'll pass the placeholder'ed data straight through to the `successHandler` but leave the cache as the API response was received so if people really want the API truth, they can hit the cache?

Reply:
More like you have the placeholder in the component that needs it.